R/DESCRIPTION

```diff
@@ -28,11 +28,14 @@ Depends:
 Imports:
     rlang,
     sparklyr (>= 1.3),
```

Comment:
In the R folder, can you try to run the following commands? Will it pass without any warning?

```
R CMD build .
R CMD check --as-cran apache.sedona_*.tar.gz
```